infrastructure/prime-app-ephemeral-template.yml

```diff
@@ -321,14 +322,31 @@ objects:
                 value: /opt/app-root/etc/certs/pharmanet-api-cert.pfx
               - name: DB_HOST
                 value: ${SVC_NAME}-postgres
-              - name: DB_NAME
-                value: prime-${SVC_NAME}
-              - name: DB_USER
-                value: prime
-              - name: DB_PASS
-                value: prime-${SVC_NAME}
+              - name: database-password
+                valueFrom:
+                  secretKeyRef:
+                    name: ${SVC_NAME}-postgres
+                    key: database-password
+              - name: database-name
+                valueFrom:
+                  secretKeyRef:
+                    name: ${SVC_NAME}-postgres
+                    key: database-name
+              - name: database-user
+                valueFrom:
+                  secretKeyRef:
+                    name: ${SVC_NAME}-postgres
+                    key: database-user
+              - name: database-password
+                valueFrom:
+                  secretKeyRef:
+                    name: ${SVC_NAME}-postgres
+                    key: database-password
               - name: POSTGRESQL_ADMIN_PASSWORD
-                value: prime-${SVC_NAME}
+                valueFrom:
+                  secretKeyRef:
+                    name: ${SVC_NAME}-postgres
+                    key: database-password
```

Comment:
this needs to point at the actual admin password.
database password is db user password. and admin password is superuser password.

Reply:
Updated in ephemeral and static templates